pkg/ovs/ovsctl/appctl.go

```diff
@@ -128,7 +124,9 @@ func (c *ovsCtlClient) runTracing(flow string) (string, error) {
 	if execErr != nil {
 		return "", execErr
 	}
-	return string(out), nil
+	// Remove "\r" to avoid format issue on Windows.
+	outStr := strings.ReplaceAll(string(out), "\r", "")
```

Comment:
I think we could use `bytes.ReplaceAll` here to avoid one cast.

Reply:
Sure, updated.